scripts/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+DIR="$(dirname "$0")"
+
+# detect docker environment we're running in
+
+# mesos
+if [ "x$MESOS_SANDBOX" != "x" ]; then
```

Comment:
is there a specific reason why using different test to check whether an environment variable exists?